CMakeLists.txt

```diff
@@ -19,6 +19,15 @@ endif()
 
 # cmake -Dtest=ON to build with tests
 option(test "Build all tests." OFF)
+option(USE_DRACO "Use Draco Compression Extension." OFF)
+
+# Draco
+if (USE_DRACO)
+  add_definitions( -DUSE_DRACO )
+  include_directories(dependencies/draco/src)
+  add_subdirectory(dependencies/draco)
```

Comment:
This should be done in the GLTF cmakelists which should be buildable as its own separate project. Right now it only builds as a child of this cmakelists. Likewise, the dependency should be moved up to GLTF/dependencies.

Reply:
It's fine for the COLLADA2GLTF library to use dependencies of the GLTF library. The problem is that right now since you don't `add_subdirectory` in the GLTF CMakeLists, but you link to draco, it won't compile on its own. 

The dependency tree should be COLLADA2GLTF->GLTF->draco where COLLADA2GLTF's draco dependency is satisfied through the GLTF dependency, and then just include the appropriate headers in the COLLADA2GLTF project.

Reply:
I see. I've moved draco to GLTF/dependencies.